devnets/1620/shared/config.yaml

```diff
@@ -9,10 +9,11 @@ CONFIG_NAME: 'lukso'
 
 # Genesis
 # ---------------------------------------------------------------
-# `2**14` (= 16,384)
-MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 110
-# Friday, 14 April 2023 13:45:00 GMT+02:00
-MIN_GENESIS_TIME: 1681731000
+# Min Genesis Validators in Genesis state
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 3000
+# Date and time (GMT): Tuesday, 9 May 2023 20:00:00
+# Shanghai time = 16128 (42 epochs in seconds) + 1683662400 + 2400 - 2
+MIN_GENESIS_TIME: 1683662400
```

Comment:
this is in the past. Set it up to today 1:40pm UTC => 1683898800 (+40min delay = 2:20pm UTC / 4:20pm berlin time)